app/views/landing-pages/parents/PageParents.vue

```diff
@@ -471,158 +389,62 @@
                 </div>
             </div>
         </div>
-
-        <backbone-modal-harness
-                ref="subscribeModal"
-                :modal-view="SubscribeModal"
-                :open="subscribeModalOpen"
-                @close="subscribeModalClosed"
-        />
     </div>
 </template>
 
 <script>
-  import { mapActions, mapState, mapGetters } from 'vuex'
-
-  import SubscribeModal from 'views/core/SubscribeModal'
-  import BackboneModalHarness from './common/BackboneModalHarness'
+  import BackboneModalHarness from '../../common/BackboneModalHarness'
+  import PageParentsSectionPremium from './PageParentsSectionPremium'
+  import PageParentsJumbotron from './PageParentsJumbotron'
 
   export default {
     components: {
       BackboneModalHarness,
```

Comment:
Do we need this on this page now that some components are refactored? Cant seem to find its usage on this component.

Reply:
good catch, removed